src/developer-guide/code-path-analysis.md

```diff
@@ -16,125 +16,125 @@ if (a && b) {
 bar();
 ```
 
-![Code Path Example](../assets/images/code-path-analysis/helo.svg)
+![代码链路示例](../assets/images/code-path-analysis/helo.svg)
 
-## Objects
+## 对象
 
-Program is expressed with several code paths.
-A code path is expressed with objects of two kinds: `CodePath` and `CodePathSegment`.
+程序可以用几个代码链路来表达。
+一个代码链路用两种对象表示：`CodePath` 和 `CodePathSegment`。
 
 ### `CodePath`
 
-`CodePath` expresses whole of one code path.
-This object exists for each function and the global.
-This has references of both the initial segment and the final segments of a code path.
+`CodePath` 表达了一个代码链路整体。
+这个对象存在于每个函数和全局中。
+它对代码链路初始段和最终段都有引用。
 
-`CodePath` has the following properties:
+`CodePath` 有以下属性：
 
-* `id` (`string`) - A unique string. Respective rules can use `id` to save additional information for each code path.
-* `origin` (`string`) - The reason that the code path was started. May be `"program"`, `"function"`, `"class-field-initializer"`, or `"class-static-block"`.
-* `initialSegment` (`CodePathSegment`) - The initial segment of this code path.
-* `finalSegments` (`CodePathSegment[]`) - The final segments which includes both returned and thrown.
-* `returnedSegments` (`CodePathSegment[]`) - The final segments which includes only returned.
-* `thrownSegments` (`CodePathSegment[]`) - The final segments which includes only thrown.
-* `currentSegments` (`CodePathSegment[]`) - Segments of the current position.
-* `upper` (`CodePath|null`) - The code path of the upper function/global scope.
-* `childCodePaths` (`CodePath[]`) - Code paths of functions this code path contains.
+* `id`（`string`） - 一个唯一的字符串。各自的规则可以使用 `id` 来保存每个代码链路额外信息。
+* `origin`（`string`） - 代码链路开始的原因。可以是 `"program"`、"function"`、`"class-field-initializer"` 或 `"class-static-block"`.
+* `initialSegment`（`CodePathSegment`） - 代码链路初始段。
+* `finalSegments`（`CodePathSegment[]`） - 包括返回和抛出的最终段。
+* `returnedSegments`（`CodePathSegment[]`） - 只包括返回的最终段。
+* `thrownSegments`（`CodePathSegment[]`） - 最后只包括抛出的片段。
+* `currentSegments`（`CodePathSegment[]`） - 当前位置的片段。
+* `upper`（`CodePath|null`） - 上层函数/全局范围的代码链路。
+* `childCodePaths`）`CodePath[]`） - 包括函数代码链路的代码链路。
 
-### `CodePathSegment`
+### `CodePathSegment` - `CodePathSegment`的代码链路
```

Comment:
Is this intentional?

Reply:
No, this should be a typo caused by copying and pasting when I was adjusting the order of words.